src/core/infer_trace.h

```diff
@@ -69,6 +69,10 @@ class InferenceTrace {
   void Report(
```

Comment:
In the contructor you need to set level_ in a way that handles the deprecated MIN and MAX settings.